### PR TITLE
Move query monitoring to overview and remove dedicate monitoring section

### LINKED ISSUE #733: Move query monitoring to main overview
Since 4.5.0 all enterprise features (including monitoring) are included in all versions of CrateDB.
Therefore I would propose to combine the former **Monitoring view** with the main **overview**

e.g.
**Alternative A**
image

**Alternative B**
image

### PR DESCRIPTION
**Pull request by proddata**
## Summary of the changes / Why this is an improvement

- Move query monitoring to overview and remove dedicate monitoring section.
- Aggregate QPS and Query duration by 10 second intervals to smoothen graph and reduce total number of points to be drawn (The former high number lead to performance problems when resizing the browser window.
- Dropped logic of buffering old results, as it not needed with the the 10 sec aggregation anymore and it lead to duplicated entries
- Slightly adjust the graph buttons to align to the right and break behaviour

closes #733

Screenshot 2021-09-19 at 16 05 31

Happy for feedback @amotl @seut @mfussenegger 

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by mfussenegger** [comment]
Seems good to me. Maybe  @belringer also has some inputs?

**Comment by proddata** [comment]
Actually I am unsure, if the "Query Per Second" and "Query Speed" isn't a bit misleading overall to show to non-superusers (`crate`), as it only includes the own queries for non-superusers.

technically this could definitely could use some clean up and deduplication (i.e. load / monitoring graphs are implemented separately)

**Review by amotl — APPROVED** [review]
Thank you for streamlining the user interface after the differentiation of the former enterprise features have been removed. As @mfussenegger also generally agrees, I believe it will be good to go.

**Comment by belringer** [comment]
Looks good to me. Since the monitoring isn't too complex and it is placed on the front page, I don't see the need for a separate section. Plus the front page is already showing monitoring related information.

**Comment by proddata** [comment]
@belringer Do you think it makes sense to make them a bit smaller and maybe display 2 or 3 graphs next to each other.

--

Plan would be to clean it up a bit more on the implementation side :)

**Comment by belringer** [comment]
@proddata Sure, maybe something like this:
image
But then I would also suggest a more refined implementation of the chart toggles, as they break a bit strangely (see screenshot above).

**Comment by proddata** [comment]
@belringer 
how about so 😬  ... would be a bit more in line with the Cloud UI

image

**Comment by proddata** [comment]
I would say it is ready for review now :)

**Review by belringer — APPROVED** [review]
Looks good to me.

**Comment by amotl** [comment]
Thank you. Let's get it merged in order to cut another release?

File: CHANGES.rst [hunk 1]
```diff
@@ -8,7 +8,13 @@ Unreleased
 - Side-Navigation: Adjusted size sligthly, added hover-effect on selected item,
   added border on active item and fixed monitoring icon
 
+<<<<<<< HEAD
 - Added ability to correctly display nested arrays in OBJECT(IGNORED) columns
+=======
+- Move Monitoring to Overview and remove dedicated section. Monitoring now 
+  always show last 15 minutes of QPS and query duration in 10 sec intervals.
+  Slighty adjusted appearance of graphs.
+>>>>>>> 504871e (Move query monitoring to overview and remove)
```
Comment (amotl) [inline CHANGES.rst:17]: It looks like conflict markers slipped in here.

File: app/static/i18n/de.json [hunk 2]
```diff
@@ -43,7 +43,9 @@
         "AFFECTED_NODES": "Betroffene Knoten",
         "INFO": "INFO",
         "WARNING": "WARNUNG",
-        "CRITICAL": "KRITISCH"
+        "CRITICAL": "KRITISCH",
+        "QUERIES_PER_SECOND": "Anfragen Pro Sekunde",
+        "QUERY_SPEED": "Abfragegeschwindigkeit (ms)"
```
Comment (amotl) [inline app/static/i18n/de.json:48]: Would this better be called "QUERY_DURATION"? Not totally sure, I would also be reasonably fine with SPEED.
```suggestion
        "QUERY_DURATION": "Abfragedauer (ms)"
```

File: CHANGES.rst (hunk 1 above)
Comment (proddata) [reply CHANGES.rst]: Thanks ... VS Code played tricks on me 😬

File: app/static/i18n/de.json (hunk 2 above)
Comment (proddata) [reply app/static/i18n/de.json:48]: I am indifferent, but would need good translations for EN,DE,FR,ES :D 
I only moved it from the monitoring plugin.

Comment (amotl) [reply app/static/i18n/de.json:48]: Then let's keep it like it was.

**Pull request merged**